### ISSUE TITLE
Can't select certain variables for comparison

### ISSUE THREAD

**Issue by macumber** [NREL/wex#61]
DView seems to get stuck in a weird state where i can select certain variables for comparison in the first column but not others

![image](https://user-images.githubusercontent.com/1327850/35010272-9881fb6e-fabf-11e7-8e31-d1cfcfb24ab8.png)

**Comment by shorowit** [comment]
For each graph, you can only select multiple variables so long as there are 2 or fewer different units across all the variables. So you can select a bunch of temperature variables (C) and a bunch of illuminance variables (lux), which results in C on the left axis and lux on the right axis. But you can't select a third type of unit (e.g., % or J) for the same graph; rather you'd have to use the second graph for that.

**Comment by macumber** [comment]
Yeah I get that, but this sometimes locks me out from selecting a variable even if there is only one or even no other variable selected

**Comment by macumber** [comment]
Also, it's intermittent, I can restart and then it is fine

**Comment by shorowit** [comment]
Can you provide a screenshot of that? The previous screenshot you attached looks fine as both C and J units are displayed, so it's correctly locking out other variables.

**Comment by macumber** [comment]
Well there was only one variable plotted, one of the axes was being used up by a variable which was not selected for plotting.
